### PR TITLE
Bug 1778290 - Disable phase schedule/signoff button once clicked

### PR DESCRIPTION
**Pull request by gabrielBusta**
Mitigates double-scheduling of phases by double-clicking on the "schedule" in the "Schedule Phase" modal.

### CODE REVIEW

**Comment by gabrielBusta** [comment]
Yeah there could be. In those cases we can refresh the page an try again. This to prevent double clicks but we can still double schedule a phase by refreshing and clicking schedule again before it is marked a submitted.

**Review by jcristau — APPROVED** [review]
Thanks!

File: frontend/src/components/PhaseProgress/index.jsx [hunk 1]
```diff
@@ -61,6 +64,8 @@ export default function PhaseProgress({ release, readOnly, xpi }) {
   };
 
   const scheduleOrSignoff = async () => {
+    setDisableScheduleOrSignoff(true);
```
Comment (JohanLorenzo) [inline frontend/src/components/PhaseProgress/index.jsx:67]: n00b question (I've never written any react code): Here we're disabling the button just before sending the request to the backend. Does react magically re-enable it later? Or the button remains disable until we recreate the modal window?

**Review by JohanLorenzo — APPROVED** [review]
Thank you very much for making the fix so fast! I'm no react expert so my review isn't worth a lot here. That said, the patch makes sense to me!

File: frontend/src/components/PhaseProgress/index.jsx (hunk 1 above)
Comment (gabrielBusta) [reply frontend/src/components/PhaseProgress/index.jsx:67]: You can re-enable it by refreshing the page. That restarts the app state. I can re-enable it after the request comes back from the back-end.

**Pull request merged**